tests/snappi/variables.py

```diff
@@ -0,0 +1,27 @@
+line_card_choice = 'chassis_multi_line_card_multi_asic'
+config_set = {
+                "chassis_single_line_card_single_asic": {
+                    'hostname': ["sonic-s6100-dut1"],
+                    'asic': ["asic0"]
+                },
+                "chassis_single_line_card_multi_asic": {
```

Comment:
I do not understand why a single S6100 switch can have two ASICs. It is a single-ASIC switch.

Reply:
this is just a variable file to check if the coressponding values are being taken, the user has to modify the asic value or the hostnames according to their testbed